Archive/Hairer.lean

```diff
@@ -32,7 +32,6 @@ variable {ι : Type*} [Fintype ι]
 section normed
 variable {𝕜 E F : Type*} [NontriviallyNormedField 𝕜]
 variable [NormedAddCommGroup E] [NormedSpace 𝕜 E] [NormedAddCommGroup F] [NormedSpace 𝕜 F]
-
```

Comment:
I think this is a case where the empty line makes sense, as the next line is actually `variable ... in`, which applies specifically to the following declaration and should not be put together with the global `variable` lines.

Reply:
This is now fixed